### DISCUSSION TITLE
Qwik-speak no longer works with Qwik@1.2.10?

### DISCUSSION THREAD

**Discussion by skhiter** [robisim74/qwik-speak#88] [Q&A]
Hello everyone,

I have encountered a problem with the installation of Qwik-speak in my Qwik project in version 1.2.10. I can no longer navigate between my pages, I see that the url changes but not the page. I'm using localized routing but the locale doesn't fit into the url anymore. I wanted to know if there was a problem with the new version of Qwik?

Best regards

**Comment by robisim74** [comment]
There is already an open issue for that: https://github.com/robisim74/qwik-speak/issues/83

I'm closing this discussion.

Greetings
